src/components/Map/Map.stories.js

```diff
@@ -12,11 +12,21 @@ export default {
   decorators: [withMapLayout],
   args: {
     markerCoordinates: [52.1326332, 5.291266],
+    defaultCenter: [52.1326332, 5.291266],
```

Comment:
changing this does not appear to have an effect in storybook -> check if it's also in the component itself but either way this should work as expected